p2p/agent.go

```diff
@@ -101,8 +101,12 @@ type (
 		lifecycle.StartStopper
 		// BroadcastOutbound sends a broadcast message to the whole network
 		BroadcastOutbound(ctx context.Context, msg proto.Message) (err error)
+		// BroadcastOutboundInBatch puts a broadcast message in batch
+		BroadcastOutboundInBatch(ctx context.Context, msg proto.Message, opts ...BatchOption) (err error)
```

Comment:
delete. `BroadcastOutbound` should be the only function to broadcast. The `opts ...BatchOption` are features of agent, not the function. One possible `opt` for the function is to broadcast immediately, e.g., for consensus component.